examples/1_Impacted_Substances_Queries/1-1_Materials_impacted_substances.py

```diff
@@ -52,9 +52,7 @@
 from ansys.grantami.bomanalytics import queries
 
 mat_query = (
-    queries.MaterialImpactedSubstancesQuery()
```

Comment:
Now I think about it, I think I formatted these other files to be narrower than what black wanted because otherwise there's a lot of horizontal scrolling in the HTML docs.

Maybe we just need to exclude the notebook python scripts from black altogether, and manage it manually.

Reply:
They _should_ be excluded, I think something weird happened with black when I updated pre-commit. I've reverted said changes, let's see if they get "fixed" again